index.js

```diff
@@ -60,6 +60,15 @@ async function generateModels(aspect, containerSelector, model, env=`live`) {
 
     // eslint-disable-next-line no-undef
     const wowModelViewer =  await new WowModelViewer(models)
+
+    // override wowhead events listeners
+    $(containerSelector + ' canvas').firstOn('dblclick', (event) => {
+        if ( window.fullScreenDisabled === true) event.stopImmediatePropagation();
+    })
+    $(containerSelector + ' canvas').firstOn('DOMMouseScroll mousewheel', (event) => {
+        if ( window.zoomDisabled === true) event.stopImmediatePropagation();
+    })
```

Comment:
I think you should put this in `class WowModelViewer` 
On a same webpage you can have multiple models.
Ex a mount and pet shop and maybe you will not want to make the same restriction on all elements
The model viewer can display:
- Characters
- NPCs
- Creatures
- Mounts
- Pets
- Items
- Game objects